packages/graphql/lib/schema-builder/metadata/enum.metadata.ts

```diff
@@ -12,4 +12,5 @@ export interface EnumMetadata<T extends object = any> {
   name: string;
   description?: string;
   valuesMap?: EnumMetadataValuesMap<T>;
+  mapToUppercase?: boolean;
```

Comment:
@kasvith this option seems to be a single purpose one. Would it be better to take a key transform function so that people can use any function they want for the enum keys? toUppercase could then become the default and the feature would become a lot more useful.

Reply:
Hi @mareksuscak

By convention GraphQL enums are UPPER_CASE literals, i dont see a usecase to have a transformer function and make this much more complicated than it is now

https://www.apollographql.com/tutorials/side-quest-intermediate-schema-design/02-the-enum-type